git-cliff-core/src/changelog.rs

```diff
@@ -142,7 +143,29 @@ impl<'a> Changelog<'a> {
 			write!(out, "{header}")?;
 		}
 		for release in &self.releases {
-			write!(out, "{}", self.template.render(release)?)?;
+			let mut rendered = self.template.render(release)?;
+			if let Some(preprocessors) =
+				self.config.changelog.postprocessors.as_ref()
+			{
+				for preprocessor in preprocessors {
+					if let Some(text) = &preprocessor.replace {
+						rendered = preprocessor
+							.pattern
+							.replace_all(&rendered, text)
+							.to_string();
+					} else if let Some(command) = &preprocessor.replace_command {
+						if preprocessor.pattern.is_match(&rendered) {
+							rendered = command::run(
+								command,
+								Some(rendered.to_string()),
+								vec![],
+							)?;
+						}
+					}
+				}
+			}
```

Comment:
I see that this is simply the same method  as `Commit::preprocess`. Can you refactor this so there isn't code duplication? One idea would be adding an helper method to somewhere, possibly in `TextProcessor` about this.

Reply:
I merged them